src/libraries/System.Security.Cryptography.Algorithms/src/System.Security.Cryptography.Algorithms.csproj

```diff
@@ -11,6 +11,8 @@
   <PropertyGroup>
     <UseAndroidCrypto Condition="'$(TargetsAndroid)' == 'true'">true</UseAndroidCrypto>
     <UseAppleCrypto Condition="'$(TargetsOSX)' == 'true' or '$(TargetsiOS)' == 'true' or '$(TargetstvOS)' == 'true'">true</UseAppleCrypto>
+    <IncludeUnavailableAppleCryptoAPIs Condition="'$(TargetsiOS)' == 'true' or '$(TargetstvOS)' == 'true'">false</IncludeUnavailableAppleCryptoAPIs>
```

Comment:
UseAppleCryptoMobile seems like a better name to me.

Reply:
Used `UseAppleCryptoMobile` instead, thanks.